pkg/syncer/k8scloudoperator/placement.go

```diff
@@ -210,15 +211,15 @@ func preFilterSPList(ctx context.Context, sps *unstructured.UnstructuredList, st
 	for _, sp := range sps.Items {
 		spName := sp.GetName()
 		spType, found, err := unstructured.NestedString(sp.Object, "metadata", "labels", spTypeLabelKey)
-		if !found || err != nil || spType != vsanDirect {
-			nonVsanDirect++
+		if !found || err != nil || (spType != vsanDirect && spType != vsanSna) {
+			nonVsanDirectOrSna++
 			continue
 		}
 
 		//sc compatible filter
 		scs, found, err := unstructured.NestedStringSlice(sp.Object, "status", "compatibleStorageClasses")
 		if !found || err != nil {
-			nonVsanDirect++
```

Comment:
this seems to be an existing bug - here we should be updating nonSCComp not nonVsanDirect - so please fix this.

Reply:
can you fix this bug? just change nonVsanDirect above to nonSCComp

Reply:
what about this @pallabi-sutradhar ?

Reply:
I already changed that.